src/inspector/view/qml/MuseScore/Inspector/common/InspectorPropertyView.qml

```diff
@@ -98,17 +98,15 @@ Column {
         Loader {
             id: buttonLoader
 
-            width: 20
-            height: width
-
             active: visible
             sourceComponent: root.isStyled ? menuButtonComponent : resetButtonComponent
 
             Component {
                 id: resetButtonComponent
 
                 FlatButton {
-                    anchors.fill: parent
+                    width: 20
```

Comment:
I'm not sure to be honest. The old version seems not to work because FlatButton has an implicit width/height which apparently breaks `anchors.fill` (but why...?). But setting the explicit width/height on the button itself does work.